server/adaptors/integrations/__data__/repository/aws_vpc_flow/aws_vpc_flow-1.0.0.json

```diff
@@ -4,7 +4,7 @@
   "displayName": "AWS VPC Flow",
   "description": "AWS VPC Flow log collector",
   "license": "Apache-2.0",
-  "type": "logs",
+  "type": "logs_vpc",
```

Comment:
all the `logs` index based integrations should use the same index template names `logs`
the only dynamic things that should be integration specific are:
 - index alias
 - `composed_of` component list

Reply:
> We discussed that we need to work with these duplicates for now to account for multiple component configs. Let's work on fixing it later, as a hacky solution I can try setting the composed_of list dynamically, but I want to keep this PR small